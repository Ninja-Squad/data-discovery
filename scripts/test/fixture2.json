[
  {
    "name": "123",
    "desc": "I'm a description!"
  },
  {
    "noname": "456",
    "desc": "I'm another description!"
  },
  {
    "name": ["789"],
    "desc": "I'm yet another description!"
  },
  {
    "name": 101112,
    "desc": "I'm still yet another description!"
  }
]