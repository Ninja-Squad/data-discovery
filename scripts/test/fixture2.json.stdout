{"index":{"_id":"123"}}
{"name":"123","desc":"I'm a description!"}
{"index":{"_id":"[\"789\"]"}}
{"name":["789"],"desc":"I'm yet another description!"}
{"index":{"_id":"101112"}}
{"name":101112,"desc":"I'm still yet another description!"}
